webpack/webpack.config.js

```diff
@@ -31,21 +31,30 @@ const externals = {
 	lodash: "window.lodash",
 };
 
-module.exports = function( env = { environment: "production", recalibration: "disabled" } ) {
+/**
+ * Helper function to get the Webpack Define plugin with an optional recalibration flag.
+ *
+ * @param {boolean} isRecalibration Whether or not the recalibration is enabled.
+ *
+ * @returns {webpack.DefinePlugin|DefinePlugin} The define plugin.
+ */
+function getDefinePlugin( isRecalibration = false ) {
```

Comment:
I would give this function a more descriptive name.
A call to `getDefinePlugin( false )` does not indicate what the function does, namely setting the recalibration flag to `false`.
I would either make it more abstract, giving it an object as an argument and implementing it like this:
```js
function createDefinePlugin( definitions ) {
     return new webpack.DefinePlugin( { "process.env": definitions } );
}
```
or give it a better name:
```js
function createRecalibrationPlugin( isRecalibration = false ) 
```